src/lib/components/forms/ManageCollaborators.svelte

```diff
@@ -0,0 +1,178 @@
+<!-- @component 
+Manage users on a project: add, update, remove
+
+This component is actually two separate forms that travel together.
+They might get separated later.
+-->
+<script lang="ts">
+  import type {
+    Project,
+    ProjectAccess,
+    ProjectUser,
+    User,
+  } from "$lib/api/types";
+
+  import { enhance } from "$app/forms";
+
+  import { createEventDispatcher } from "svelte";
+  import { _ } from "svelte-i18n";
+
+  import Button from "../common/Button.svelte";
+  import Field from "../inputs/Field.svelte";
+  import Flex from "../common/Flex.svelte";
+  import Text from "../inputs/Text.svelte";
+
+  import { canonicalUrl } from "$lib/api/projects";
+
+  export let project: Project;
+  export let users: ProjectUser[];
+
+  const dispatch = createEventDispatcher();
+
+  $: canonical = canonicalUrl(project);
+  $: action = new URL("?/users", canonical).href;
+  $: invite = new URL("?/invite", canonical).href;
+  $: grouped = users.reduce(
+    (m, { user, access }) => {
+      m[access].push(user);
+      return m;
+    },
+    { admin: [], edit: [], view: [] } satisfies Record<ProjectAccess, User[]>,
+  );
+
+  function sort(users: User[]) {
+    return users.sort((a, b) => a.name.localeCompare(b.name));
+  }
+
+  function onSubmit({ formElement, formData, action, cancel, submitter }) {
+    submitter.disabled = true;
+    return async ({ result, update }) => {
+      submitter.disabled = false;
+      return update(result);
+    };
+  }
+</script>
+
+<Flex direction="column">
+  <form action={invite} method="post" use:enhance>
+    <h2>{$_("collaborators.addCollaborators")}</h2>
+    <p>{@html $_("collaborators.invite")}</p>
+    <Flex align="center">
+      <Field title={$_("common.emailAddress")} sronly>
+        <Text name="email" placeholder={$_("common.emailAddress")} />
+      </Field>
+      <Field>
+        <select name="access" class="access">
+          <option value="view">{$_("collaborators.view")}</option>
+          <option value="edit">{$_("collaborators.edit")}</option>
+          <option value="admin">{$_("collaborators.admin")}</option>
+        </select>
+      </Field>
+      <Button mode="primary" type="submit">
+        + {$_("collaborators.add")}
```

Comment:
Since the plus is decorative here, we should prefer an icon: `Plus16` from `svelte-octicons` should do the trick.